src/GameComponent/Game.jsx

```diff
@@ -0,0 +1,429 @@
+import React, { Component } from "react";
+import CountDownTimer from "../timerCounter/Countdown";
+import data from "../data/dictionary.json";
+import "./Game.css";
+class startScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.currentState = {
```

Comment:
Any variable/property that doesn't change based on user action shouldn't be part of the state object.
For example, `easyWords`, `mediumWords`, and `hardWords` does not depend on user's action.